src/clientiface.cpp

```diff
@@ -196,6 +196,20 @@ void RemoteClient::GetNextBlocks (
 	s16 wanted_range = sao->getWantedRange() + 1;
 	float camera_fov = sao->getFov();
 
+	const s16 full_d_max = std::min(adjustDist(m_max_send_distance, camera_fov), wanted_range);
+	const s16 d_opt = std::min(adjustDist(m_block_optimize_distance, camera_fov), wanted_range);
+	const s16 d_blocks_in_sight = full_d_max * BS * MAP_BLOCKSIZE;
+	infostream << "Fov from client " << camera_fov << " full_d_max " << full_d_max << std::endl;
+
+	s16 d_max = full_d_max;
+	s16 d_max_gen = std::min(adjustDist(m_max_gen_distance, camera_fov), wanted_range);
+
+	// Don't loop very much at a time, adjust with distance,
+	// do more work per RTT with greater distances.
+	s16 max_d_increment_at_time = full_d_max / 10 + 1;
```

Comment:
For normal view at default settings i guess this will be 1?
9 / 10 rounded to 0, + 1
DOes 1 work better than the previous 2?

Reply:
Yeah, for small distances 1 works better, for very short 0 is better still. 9 is kinda at the border, though. Could also divide by 9... Let me test this specifically.

Reply:
No problem, i just seemed to remember some concern about whether 2 is actually better than 1, so just checking, i'll leave this to you to decide.